packages/client/src/client.js

```diff
@@ -190,13 +190,28 @@ export default class PostRPCClient {
 
 	/**
 	 * Call a registered RPC
-	 * @param {String} method
-	 * @param {Object|Array} params
-	 * @param {Function} callback to return response
-	 * @param {Number} timeout in MS to await response
-	 * @return {Undefined}
+	 * @param {{
+	 * 	method: String
+	 * 	args?: Object|Array
+	 * 	callback?: Function
+	 * 	timeout?: Number
+	 * }} details
+	 * @returns {Undefined}
 	*/
-	call (method, params, callback = null, timeout = 5000) {
+	call (details) {
+		if (!details) {
```

Comment:
i think should check that details is an object here, containing a method key. because when i pulled this code and ran it against the old way of calling it, i.e. `client.call('resize', { dimensions })`, I was getting an opaque error - 
`{code: -32602, message: "Invalid args", data: "Invalid method argument(s)"}`

Reply:
agree.